src/org/opendatakit/briefcase/export/XmlElement.java

```diff
@@ -174,6 +193,16 @@ public String getValue() {
         .filter(s -> !s.isEmpty());
   }
 
+  /**
+   * Returns true if this element has an attribute with the given name, false otherwise
```

Comment:
`Returns whether this element has an attribute with the given name`
would be shorter, if you’re interested

Reply:
I'd naturally write the "whether/or" sentence, but I tend to see the "Returns x if y. z otherwise" in many Java classes...

Reply:
If Josh Bloch wrote it, I’d probably emulate it. Otherwise, I’d be my own man. Up to you.